vortexasdk/endpoints/freight_pricing_result.py

```diff
@@ -97,11 +97,10 @@ def to_df(self, columns=None) -> pd.DataFrame:
 
 
 DEFAULT_COLUMNS = [
-    'short_code',
-    'rate'
-    'rate_unit',
-    'cost',
-    'cost_unit',
-    'tce',
-    'tce_unit'
+    "short_code",
+    "rate" "rate_unit",
```

Comment:
This is a bug

Reply:
Fixed in my other PR, but also fixed here now.